resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "fundReleaseId": "3456789001",
+          "eventDate": "2017-05-10",
+          "eventType": "Extension one"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "LIFE_EVENT_ALREADY_EXISTS",
+          "message": "The investor’s life event has already been reported",
+          "lifeEventId": "6789000001"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The associated fund release has been superseded</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                            <br>
+                            accountId: 2000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "fundReleaseId": "3456789000",
+          "eventDate": "2017-05-10",
+          "eventType": "Extension one"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "FUND_RELEASE_SUPERSEDED",
+          "message": "This fund release has already been superseded"
+          "lifeEventId": "3456789001"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The extension you are superseding has already been superseded</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                            <br>
+                            accountId: 1000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "eventDate": "2017-05-11",
+          "eventType": "Extension one",
+          "supersede": {
+            "originalEventDate": "2017-05-10",
+            "originalLifeEventId": "6789000001"
+          }
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+        <pre class="code--block">
+        {
+          "code": "SUPERSEDED_LIFE_EVENT_ALREADY_SUPERSEDED",
+          "message": "This life event has already been superseded",
+          "lifeEventId": "6789000002"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: requestPropertyPurchaseExtension
+      parameters:
```

Comment:
This is also missing a Content-Type header (the RAML is incorrect) as shown in the snippet below.

<img width="637" alt="Screenshot 2023-02-09 at 11 14 10" src="https://user-images.githubusercontent.com/63714619/217836735-df4d47f7-02f5-4694-b9c7-556900faa033.png">

Please correct this at least in this PR by adding the reference `- $ref: '#/components/parameters/contentTypeHeader'` as making this endpoint call without the header results in a 400. Thanks. This is shown in the snippets below

<img width="961" alt="Screenshot 2023-02-09 at 14 17 32" src="https://user-images.githubusercontent.com/63714619/217837769-1e9a41fc-4418-46b2-b080-ae6ffb0eb1fa.png">

<img width="962" alt="Screenshot 2023-02-09 at 14 17 16" src="https://user-images.githubusercontent.com/63714619/217837878-6a8e5c98-1bac-45fe-b82f-95ffb538b573.png">